lib/ui/views/home/home_view.dart

```diff
@@ -110,25 +91,36 @@ class _HomeViewState extends State<HomeView> {
               SliverList(
                 delegate: SliverChildListDelegate([
                   SizedBox(height: 15),
-                  QuickSearch(),
+                  model.isBusy
+                      ? Padding(
+                          padding: const EdgeInsets.symmetric(horizontal: 10.0),
+                          child: ShimmerGrid())
+                      : QuickSearch(),
                 ]),
               ),
               SliverPadding(
                 padding: EdgeInsets.symmetric(horizontal: 10),
                 sliver: SliverList(
                   delegate: SliverChildListDelegate([
                     SizedBox(height: 5),
-                    AnnouncementCarousel(model.announcements),
+                    model.isBusy
+                        ? ShimmerList()
+                        : AnnouncementCarousel(model.announcements),
                   ]),
                 ),
               ),
               SliverList(
-                delegate: SliverChildListDelegate([
-                  ResultsListView('Nearest to You', model.nearestPlaces),
-                  ResultsListView('Recommended', model.recommendedPlaces),
-                  ResultsListView('Popular among users', model.popularPlaces),
-                  ResultsListView('Budget options', model.budgetPlaces),
-                ]),
+                delegate: model.isBusy
+                    ? SliverChildListDelegate([
```

Comment:
I think this part of the code can be better structured in this way: 

`model.isBusy
                  ? SliverToBoxAdapter(child: SizedBox())
                  : SliverList(
                      delegate: SliverChildListDelegate([
                        ResultsListView('Nearest to You', model.nearestPlaces),
                        ResultsListView('Recommended', model.recommendedPlaces),
                        ResultsListView(
                            'Popular among users', model.popularPlaces),
                        ResultsListView('Budget options', model.budgetPlaces),
                      ]),
                    ),
            ]),`

Reply:
sure, make the commit then